plugins/speculation-rules/helper.php

```diff
@@ -103,7 +103,7 @@ static function ( string $href_exclude_path ) use ( $prefixer ): string {
 					// Also exclude rel=nofollow links, as plugins like WooCommerce use that on their add-to-cart links.
 					array(
 						'not' => array(
-							'selector_matches' => 'a[rel=nofollow]',
+							'selector_matches' => 'a[rel*=nofollow]',
```

Comment:
I think the "attribute contains" selector is more appropriate here since we want to match the entire token. Also, might as well quote it.

```suggestion
							'selector_matches' => 'a[rel~="nofollow"]',
```